src/frequenz/channels/_anycast.py

```diff
@@ -68,10 +68,19 @@ def __init__(self, maxsize: int = 10) -> None:
             maxsize: Size of the channel's buffer.
         """
         self.limit: int = maxsize
+        """The maximum number of items that can be stored in the channel's buffer."""
```

Comment:
?
```suggestion
        """The maximum number of messages that can be stored in the channel's buffer."""
```

Reply:
btw `self.limit` is no longer used/needed so it can actually be deleted. Only `maxsize` is used to initialize the channel's buffer. I'll remove it if there is no objections.
And I'd extend the documentation for the blocking conditions instead

Reply:
Added a new commit to remove `self.limit`